src/Uno.UWP/Devices/Midi/MidiInPort.wasm.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Uno.Devices.Enumeration.Internal;
+
+namespace Windows.Devices.Midi
+{
+    public partial class MidiInPort
+    {
+		private const string JsType = "Windows.Devices.Midi.MidiInPort";
+		private readonly string _wasmId;
+
+		private readonly static ConcurrentDictionary<string, MidiInPort> _instanceSubscriptions =
+			new ConcurrentDictionary<string, MidiInPort>();
+
+		private MidiInPort(string deviceId, string wasmId)
+		{
+			DeviceId = deviceId;
+			_wasmId = wasmId;
+		}
+
+		public static int DispatchMessage(string serializedMessage)
+		{
+			var splitMessage = serializedMessage.Split(':');
+			var timestamp = TimeSpan.FromMilliseconds(double.Parse(splitMessage[0], CultureInfo.InvariantCulture));
+			var message = new byte[splitMessage.Length - 1];
+			for (int i = 1; i < splitMessage.Length; i++)
+			{
+				message[i-1] = byte.Parse(splitMessage[i]);
+			}
+
+
+			return 0;
+		}
+
+		public void Dispose()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either implement 'IDisposable.Dispose', or totally rename this method to prevent confusion.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)